src/components/SpaceProposalBoostItem.vue

```diff
@@ -0,0 +1,465 @@
+  const formattedUnits = formatUnits(
+    props.boost.currentBalance,
+    props.boost.token.decimals
+  );
+  return formatNumber(
+    Number(formattedUnits),
+    getNumberFormatter({ maximumFractionDigits: 8 }).value
+  );
+});
+
+const amountPerWinner = computed(() => {
+  const isFinalAndLessVotes =
+    Number(props.boost.strategy.distribution.numWinners) >
+      Number(props.proposal.votes) && isFinal.value;
+
+  const amountOfWinners = isFinalAndLessVotes
+    ? Number(props.proposal.votes)
+    : Number(props.boost.strategy.distribution.numWinners);
+
+  const amount = Number(boostBalanceFormatted.value) / amountOfWinners;
+
+  return `${formatNumber(
+    Number(amount),
+    getNumberFormatter({ maximumFractionDigits: 8 }).value
+  )} ${props.boost.token.symbol}`;
+});
+
+const claimPeriodEnded = computed(() => {
+  return Number(props.boost.end) < Date.now() / 1000;
+});
+
+const isOwner = computed(() => {
+  return (
+    toChecksumAddress(props.boost.owner) ===
+    toChecksumAddress(web3Account.value)
+  );
+});
+
+const isLottery = computed(() => {
+  return props.boost.strategy.distribution.type === 'lottery';
+});
+
+const lotteryNoRewardFinal = computed(() => {
+  return isLottery.value && !props.reward && isFinal.value;
+});
+
+const lotteryLimit = computed(() => {
+  if (!isLottery || !props.boost.strategy.distribution.limit) return 0;
+  return Number(props.boost.strategy.distribution.limit) / 100;
+});
+
+const weightedLimit = computed(() => {
+  if (isLottery.value || !props.boost.strategy.distribution.limit) return 0;
+  return formatUnits(
+    props.boost.strategy.distribution.limit,
+    props.boost.token.decimals
+  );
+});
+
+const boostNetworkInfo = computed(() => {
+  return networks?.[props.boost.chainId];
+});
+
+async function withdraw(boost: BoostSubgraph) {
+  try {
+    const tx = await withdrawAndBurn(
+      auth.web3,
+      boost.chainId,
+      boost.id,
+      web3Account.value
+    );
+    await tx.wait();
+    emit('reload');
+  } catch (e) {
+    console.error('Error withdrawing boost', e);
+  }
+}
+
+async function loadWinners() {
+  try {
+    const response = await getWinners(
+      props.boost.strategy.proposal,
+      props.boost.id,
+      props.boost.chainId
+    );
+
+    if (response.winners.length > 0) {
+      lotteryWinners.value = response.winners;
+      lotteryPrize.value = response.prize;
+      // TODO: Figure out reload
+      // if (!props.reward) window.location.reload();
+    }
+    return response.winners;
+  } catch (e) {
+    console.error('Error loading winners', e);
+  }
+}
+
+watch(
+  isFinal,
+  async () => {
+    if (isFinal.value && isLottery.value) {
+      await loadWinners();
+      if (lotteryWinners.value.length) return;
+
+      const interval = setInterval(async () => {
+        const winners = await loadWinners();
+
+        if (winners && winners.length > 0) {
+          clearInterval(interval);
+        }
+      }, 60000);
+      onUnmounted(() => {
```

Comment:
This is creating the following warning:

![Screenshot 2024-03-07 at 00 49 07](https://github.com/snapshot-labs/snapshot/assets/495709/8e36fca6-b307-444b-bc62-d94210fffbf0)

When the proposal have lottery boosts without any winners.